Gemfile.lock

```diff
@@ -119,4 +121,4 @@ DEPENDENCIES
   rouge (~> 2.0.5)
 
 BUNDLED WITH
-   1.14.3
+   1.15.0
```

Comment:
This file was automatically modified by Ruby on my machine. Do the changes affect anything?